module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,60 @@
+#include "compression.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <utility>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(
+    const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    std::vector<std::pair<uint8_t, uint8_t>> compressed{};
+    compressed.reserve(height * width);
+    int occurrences = 0;
```

Comment:
I would use `uint8_t` for consistency (pair has uint_8_t) so int is superfluous.

Reply:
Thanks, done :)